src/librustc_incremental/calculate_svh/hasher.rs

```diff
@@ -9,13 +9,16 @@
 // except according to those terms.
 
 use std::mem;
+use std::hash::Hasher;
 use rustc_data_structures::blake2b::Blake2bHasher;
 use rustc::ty::util::ArchIndependentHasher;
 use ich::Fingerprint;
+use rustc_serialize::leb128::write_unsigned_leb128;
 
 #[derive(Debug)]
 pub struct IchHasher {
     state: ArchIndependentHasher<Blake2bHasher>,
+    bytes: Vec<u8>,
```

Comment:
Can you rename this to something like `leb128_helper`? Otherwise one might think that it contains the bytes that were hashed so far.